helm_deploy/calculate-journey-variable-payments/templates/cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "app.fullname" . }}
+  labels:
+    app: {{ template "app.name" . }}
+    chart: {{ template "app.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+
+spec:
+  schedule: "0 3 */1 * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: calculate-journey-variable-payments
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              command: ["java", "-jar", "/app/app.jar", "--import-locations-and-prices" ]
```

Comment:
My fault this one, ticket not very clear.  Should start with `--import-supplier-reports`

Reply:
changed